build/contracts.json

```diff
@@ -12,6 +12,7 @@
         "name": "distribute",
         "outputs": [],
         "payable": false,
```

Comment:
Why do you have a json file with generated abi commited in git?

Reply:
We currently use it in: 
https://github.com/raiden-network/raiden-token/blob/6225171f40fc4dc332218f15d670b2b36d0271d3/event_sampler/main.py#L47

I know populus has `chain.provider.get_base_contract_factory(contract_name).abi`. We will switch to this.

Can this be merged, or do you want me to remove it now?

Reply:
Please remove it if possible, it makes no sense to update something that is automatically generated by hand.